Gemfile

```diff
@@ -19,12 +19,82 @@ source 'https://rubygems.org'
 
 gemspec :name => "chef-dk"
 
+# EXPERIMENTAL: ALL gems specified here will be installed in chef-dk omnibus.
+# This represents all gems that will be part of chef-dk.
+
+# All software we recognize needs to stay at the latest possible version. But
+# since that's not expressible here, we make it >= the last *known* version to
+# at least prevent downgrades beyond that:
+gem "appbundler", github: "chef/appbundler", branch: "jk/multiple-gems"
+gem "berkshelf"
 # Chef 12.8.1 Gem includes some extra files which can break gem installation on
 # windows. For now we are pulling chef from github at the tag as a workaround.
```

Comment:
we should delete this comment too since we are pulling from GH for different reasons